### ISSUE TITLE
OpenSimplexNoise not declared in the current scope

### ISSUE THREAD

**Issue by thechubbypanda** [godotengine/godot#61059]
### Godot version

v4.0.alpha [e8520044e]

### System information

ArchLinux (godot-git)

### Issue description

OpenSimplexNoise doesn't appear to be available in GDScript, it just throws the error in the title in editor and when running.
I'm probably doing something stupid but I've followed the latest docs and it's just not having it.
Has it been removed or something?

### Steps to reproduce

- Build latest Godot
- Create new project
- Create script
- `@onready var noise = OpenSimplexNoise.new()`
- Observe error

### Minimal reproduction project

[Uploading OSN_Test.zip…]()

**Comment by clayjohn** [comment]
Yes. OpenSimplexNoise has been removed in favour of FastNoiseLite. See more in the PR here https://github.com/godotengine/godot/pull/56718

The online docs still need to be updated I guess.
